### PR TITLE
Add categories

### PR DESCRIPTION
**Pull request by JelenadlS**

### CODE REVIEW

**Review by F-Kirchhoff — CHANGES_REQUESTED** [review]
Hey nice work! Just found a little problem with your urls.

File: src/components/Form.js [hunk 1]
```diff
@@ -44,7 +60,22 @@ export default function Form({ handleActivity, preloadedValues, title }) {
...
+                  const secondFriend = b.toLowerCase();
+
+                  if (firstFriend < secondFriend) return -1;
+                  if (firstFriend > secondFriend) return 1;
+                  return 0;
+                })
+                .join(', '),
         notes: data.notes,
         date: data.date,
         location: data.location,
```
[the remark this answers is not in the document]
Comment (JelenadlS) [reply src/components/Form.js:81]: oh wow, thats a good idea😅🙈
fixed😊

File: src/pages/ActivityOverviewPage.js [hunk 2]
```diff
@@ -28,53 +38,87 @@ export default function ActivityOverviewPage({ activities }) {
...
-            navigate(
-              `/${selectedActivity.friend}/${selectedActivity.activity}/${selectedActivity.id}/editactivity`
-            )
-          }
-        >
-          <img src={editIcon} alt="edit" />
-        </EditButton>
+        <MainGrid>
+          <StyledTitle>
+            <strong>Activity</strong>
```
[the remark this answers is not in the document]
Comment (JelenadlS) [reply src/pages/ActivityOverviewPage.js:43]: Thank you! completely oversaw it due to all the changes😊

File: src/pages/ActivityOverviewPage.js [hunk 3]
```diff
@@ -28,53 +38,87 @@ export default function ActivityOverviewPage({ activities }) {
...
+            ) : (
+              ''
+            )}
+            {selectedActivity.category === 'other' ? (
+              <img src={otherIcon} alt="other" />
+            ) : (
+              ''
+            )}
+            {selectedActivity.category}
+          </StyledCategory>
```
[the remark this answers is not in the document]
Comment (JelenadlS) [reply src/pages/ActivityOverviewPage.js:75]: I thought its interesting, so I tried it out - now per accident I figured a really short version out, I dont think its switch, but its really nice working 😅- Thank you! 🤩

File: src/pages/ActivityOverviewPage.spec.js [hunk 4]
```diff
@@ -3,44 +3,108 @@ import { render, screen } from '@testing-library/react';
 
 import ActivityOverviewPage from './ActivityOverviewPage';
 
+jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'),
+  useParams: () => ({
+    friendsName: 'Clara',
+    activityName: 'Frau Möller',
+  }),
+  useRouteMatch: () => ({ url: '/Frau Möller' }),
```
[the remark this answers is not in the document]
Comment (JelenadlS) [reply src/pages/ActivityOverviewPage.spec.js:12]: True, useNavigate also works 👍🏼
Well and good to know whit the Url rules... I texted you private, as im actually not really able to fix it. However, I would then fix it in an extra user Story..

File: src/components/Form.js [hunk 5]
```diff
@@ -135,7 +122,7 @@ export default function Form({ handleActivity, preloadedValues, title }) {
       <label htmlFor="friend">
         who should join you?
         <div>
-          <i>*Please seperate by comma when more than one friend*</i>
+          <i>*Seperate by comma when more than one friend*</i>
```
Comment (ahohnsen) [inline src/components/Form.js:125]: I think this should be "separate"

**Review by ahohnsen — APPROVED** [review]
Nice refactoring :) looks all good to me. Just a small typo!

**Pull request merged**